### PR TITLE
Fix Issue 17369 - [Module std.traits] Documentation lists ditto in table

### PR DESCRIPTION
**Pull request by RazvanN7**

### CODE REVIEW

File: std/traits.d [hunk 1]
```diff
@@ -279,21 +279,21 @@ private
 
 package
 {
-    /// Add specific qualifier to the given type T.
+    /// Add the mutable qualifier to the given type T.
     template MutableOf(T)     { alias MutableOf     =              T  ; }
```
Comment (wilzbach) [inline std/traits.d]: This is `package` and shouldn't be displayed on the docs ... 

DScanner issue: https://github.com/dlang-community/D-Scanner/issues/493

Comment (RazvanN7) [inline std/traits.d]: done

**Comment by wilzbach** [comment]
I fixed your typo in the commit message it's `MutableOf`, not `ConstOf` ;-)

Regarding the CircleCi failure, see https://github.com/dlang/phobos/pull/5559

**Comment by RazvanN7** [comment]
Thanks for the fix!

**Comment by JackStouffer** [comment]
@wilzbach looks like there's an error with dscanner.

**Comment by wilzbach** [comment]
> @wilzbach looks like there's an error with dscanner.

It's libdparse, but yep, aware of it -> https://github.com/dlang/phobos/pull/5559
(without #5559 the fix is to rebase, to latest, greatest `upstream/master`)

**Pull request merged**